app/integration_action.go

```diff
@@ -228,16 +228,18 @@ func (a *App) DoActionRequest(rawURL string, body []byte) (*http.Response, *mode
 }
 
 func (a *App) OpenInteractiveDialog(request model.OpenDialogRequest) *model.AppError {
-	clientTriggerId, userId, err := request.DecodeAndVerifyTriggerId(a.AsymmetricSigningKey())
-	if err != nil {
-		return err
+	if request.UserId == "" {
```

Comment:
Shouldn't this be `request.TriggerId != ""`?

Reply:
I am 0/5 for either being better than the other, so I changed it to what you requested.  :)